mappings/net/minecraft/item/Item.mapping

```diff
@@ -33,10 +33,10 @@ CLASS net/minecraft/class_1792 net/minecraft/item/Item
 		ARG 3 slot
 		ARG 4 clickType
 		ARG 5 player
-		ARG 6 commandItemSlot
+		ARG 6 cursorSlot
 	METHOD method_31567 isItemBarVisible (Lnet/minecraft/class_1799;)Z
 		ARG 1 stack
-	METHOD method_31568 hasStoredInventory ()Z
+	METHOD method_31568 canBeStoredInItems ()Z
```

Comment:
This method specifically is used for determining whether an item can be placed into a bundle (an item) or shulker box slot (not an item!). As such, I would probably rename it to something more generic and add documentation listing those two usages.

Reply:
I think it's reasonable to call a shulker box an item, as its primarily a storage block that can be picked up, but I am open to name suggestions for this one. When I originally brought this up in #yarn this was the name people seemed most happy with.

Reply:
Maybe `canBeNested`? Something like java's nested class concepts

Reply:
I think that's okay, but I find it a bit vague, I'll go with it if we can't find something better though

Reply:
I think that `canBeNested` would work as long as it is supplemented by documentation.